src/parse-nm.c

```diff
@@ -697,6 +699,14 @@ netplan_parser_load_keyfile(NetplanParser* npp, const char* filename, GError** e
         }
     }
 
+    /* Handle VRFs */
+    if (nd_type == NETPLAN_DEF_TYPE_VRF) {
+        if (g_key_file_has_key(kf, "vrf", "table", NULL)) {
+            nd->vrf_table = g_key_file_get_integer(kf, "vrf", "table", NULL);
+        }
+        _kf_clear_key(kf, "vrf", "table");
```

Comment:
nitpick (non-blocking): This can probably move into the `if (g_key_file_has_key(kf, "vrf", "table", NULL))` case above, as we don't need to clear a non-existing field.